src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java

```diff
@@ -244,13 +246,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         initializeDispCalListPreference();
         findPreference(mDispCalSwitchPref).setOnPreferenceChangeListener(mPreferenceListener);
 
-        String adbN = SystemProperties.get(PREF_ADB_NETWORK_READ);
-        boolean adbNB = isNumeric(adbN) && (Integer.parseInt(adbN) > 0);
-        mPrefEditor.putBoolean(mADBOverNetworkSwitchPref, adbNB);
-        SwitchPreference adbon = (SwitchPreference) findPreference(mADBOverNetworkSwitchPref);
-        // Set the switch state accordingly to the Preference
-        adbon.setChecked(adbNB);
-        updateADBSummary(adbNB);
+        mUserManager = (UserManager) getSystemService(Context.USER_SERVICE);
```

Comment:
You can avoid the cast and use
getSystemService(UserManager.class)

Reply:
Done